webspark_isearch.module

```diff
@@ -17,3 +17,25 @@ function webspark_isearch_theme_suggestions_block_alter(array &$suggestions, arr
     }
   }
 }
+
+/*
+ * Implements hook_preprocess_block().
+ */
+function webspark_isearch_preprocess_block(&$variables) {
+  // Check if we have a block entity.
+  if (isset($variables['content']['#block_content']) && ($block = $variables['content']['#block_content'])) {
+    // Split the asurite id for the react component.
+    if (isset($block->field_ids->value) && $azurite_id = $block->field_ids->value) {
```

Comment:
Let's use `$asurite_id` with an S instead of a Z to be in line with the correct spelling.